src/Core/Backup/Exception/BackupException.php

```diff
@@ -26,11 +26,11 @@
 
 namespace PrestaShop\PrestaShop\Core\Backup\Exception;
 
-use Exception;
+use PrestaShop\PrestaShop\Core\Exception\CoreException;
 
 /**
  * Class BackupException is thrown whenever backup fails.
  */
-class BackupException extends Exception implements ExceptionInterface
+class BackupException extends CoreException implements ExceptionInterface
```

Comment:
We have a backup `ExceptionInterface` ? 😄

Reply:
Isn't it a little overkill?

Reply:
Yes I was surprised too. I think `ExceptionInterface` is totally useless.

Reply:
I let you decide if this should be removed, or if we continue as-is.